simulations/Diff_Drive/read_me.txt

```diff
@@ -0,0 +1,3 @@
+step 1: to initialize the simulink profile you need to run the first block in plotScript.mlx
+step 2: run the simulink profile diff_drive.slx
+step 3: run the second part of the plotScript.mlx to get the rest of the plots
```

Comment:
Rename this file to `README.txt`. With that name GitHub automatically renders the file if we enter `simulations/`.